librecad/src/ui/forms/qg_commandwidget.cpp

```diff
@@ -189,31 +190,27 @@ void QG_CommandWidget::setNormalMode() {
 }
 
 QString QG_CommandWidget::getRootCommand( const QStringList & cmdList, const QString & typed ) {
-    QString shortestString;
-    int lengthShortestString(0);
-
-    shortestString = cmdList.at(0);
-    lengthShortestString = shortestString.length();
-
-    for(QStringList::const_iterator it = cmdList.begin(); it != cmdList.end(); ++it) {
-        if((*it).length() < lengthShortestString) {
-            shortestString = (*it);
-            lengthShortestString = shortestString.length();
-        }
-    }
-
-    // Now we parse the cmdList list, character of each item by character.
-    bool common = true;
-    int low = typed.length();
-    int high = lengthShortestString + 1;
-    int mid(0);
-    QString proposal;
+	//do we have to check for empty cmdList?
+	if(cmdList.empty()) return QString();
```

Comment:
I don't think we have to check this, as `getRootCommand` is only called if reducedChoice > 0 (in fact > 1).